docs/misc/tools/minimessage-web-editor.md

```diff
@@ -0,0 +1,11 @@
+---
+slug: /tools/minimessage-web-editor
+description: A web-based editor for creating and previewing MiniMessage-formatted text.
```

Comment:
the front matter title could exclude the heading's parentheses, looks a little nicer on the sidebar
```suggestion
title: MiniMessage Web Editor
description: A web-based editor for creating and previewing MiniMessage-formatted text.
```